cosmoboost/lib/KernelODE.py

```diff
@@ -26,13 +27,74 @@ def dK_deta(Kstore, eta, Bmatrix):
 
     return K_return
 
+def est_K_T_ODE(pars, save_kernel=True, rtol=1.e-3, atol=1.e-6, mxstep=0):
```

Comment:
This is great! 

Can you remove ` rtol=1.e-3, atol=1.e-6, mxstep=0`? We don't need it here because we're not using the scipy.odeint function anymore.